buildx/working-with-buildx.md

```diff
@@ -14,24 +14,42 @@ multiple nodes concurrently.
 
 ## Install
 
-Docker Buildx is included in Docker Desktop and Docker Linux packages when
-installed using the [DEB or RPM packages](../engine/install/index.md).
+### Windows and macOS
 
-You can also download the latest `buildx` binary from the
-[Docker buildx](https://github.com/docker/buildx/releases/latest){:target="_blank" rel="noopener" class="_"} releases page
-on GitHub, copy it to `~/.docker/cli-plugins` folder with name
-`docker-buildx` and change the permission to execute:
+Docker Buildx is included in [Docker Desktop](https://www.docker.com/products/docker-desktop){:target="_blank" rel="noopener" class="_"}
+for Windows and macOS.
 
-```console
-$ chmod a+x ~/.docker/cli-plugins/docker-buildx
+### Linux packages
+
+Docker Linux packages also include Docker Buildx when installed using the
+[DEB or RPM packages](../engine/install/index.md).
+
+### Manual download
+
+You can also download the latest binary from the [releases page on GitHub](https://github.com/docker/buildx/releases/latest){:target="_blank" rel="noopener" class="_"}.
+
+Rename the relevant binary for your OS to `docker-buildx` and copy it to
+`$HOME/.docker/cli-plugins`.
+
+Or copy it into one of these folders for installing it system-wide:
+
+* `/usr/local/lib/docker/cli-plugins` OR `/usr/local/libexec/docker/cli-plugins`
+* `/usr/lib/docker/cli-plugins` OR `/usr/libexec/docker/cli-plugins`
+
+Might require to make it executable with `chmod +x`:
+
+```shell
+$ chmod +x ~/.docker/cli-plugins/docker-buildx
 ```
 
-Here is how to use buildx inside a Dockerfile through the
+### Dockerfile
+
+Here is how to install and use Buildx inside a Dockerfile through the
 [`docker/buildx-bin`](https://hub.docker.com/r/docker/buildx-bin) image:
 
 ```dockerfile
 FROM docker
-COPY --from=docker/buildx-bin /buildx /usr/libexec/docker/cli-plugins/docker-buildx
+COPY --from=docker/buildx-bin:latest /buildx /usr/libexec/docker/cli-plugins/docker-buildx
 RUN docker buildx version
```

Comment:
Should this (and other instructions) include a `docker buildx install` step, and a way to verify that `docker buildx install` correctly replaced `docker build` to be backed by buildx?

Reply:
We already have a dedicated section below that talks about it `Set buildx as the default builder`. We might update it when https://github.com/docker/cli/pull/3314 is merged/released.